integration-tests/tests/src/tests/sync/mixed.ts

```diff
@@ -137,78 +209,65 @@ function describeRoundtrip({
 
       expect(typeof obj).equals("object");
       // Test the single value
-      performTest(obj.value, this.value);
+      performTest(obj.value, this.value, this.realm);
       // Test the list of values
       expect(obj.list.length).equals(4);
       const firstElement = obj.list[0];
-      performTest(firstElement, this.value);
+      performTest(firstElement, this.value, this.realm);
       // No need to keep these around
       delete this._id;
       delete this.value;
     });
   });
 }
 
-function describeTypes(flexibleSync: boolean) {
+function describeTypes(useFlexibleSync: boolean) {
   authenticateUserBefore();
 
-  describeRoundtrip({ typeName: "null", value: null, flexibleSync });
+  describeRoundtrip({ typeName: "null", value: null, useFlexibleSync });
 
-  // TODO: Provide an API to speficy storing this as an int
-  describeRoundtrip({ typeName: "int", value: 123, flexibleSync });
+  // TODO: Provide an API to specify storing this as an int
+  describeRoundtrip({ typeName: "int", value: 123, useFlexibleSync });
 
   // TODO: Provide an API to specify which of these to store
-  describeRoundtrip({ typeName: "float / double", value: 123.456, flexibleSync });
-
-  describeRoundtrip({ typeName: "bool (true)", value: true, flexibleSync });
-  describeRoundtrip({ typeName: "bool (false)", value: false, flexibleSync });
-
-  describeRoundtrip({ typeName: "string", value: "test-string", flexibleSync });
-
-  // Unsupported:
-  // describeSimpleRoundtrip("undefined", undefined);
+  describeRoundtrip({ typeName: "float / double", value: 123.456, useFlexibleSync });
+  describeRoundtrip({ typeName: "bool (true)", value: true, useFlexibleSync });
+  describeRoundtrip({ typeName: "bool (false)", value: false, useFlexibleSync });
+  describeRoundtrip({ typeName: "string", value: "test-string", useFlexibleSync });
 
   const buffer = new Uint8Array([4, 8, 12, 16]).buffer;
   describeRoundtrip({
     typeName: "data",
     value: buffer,
-    testValue: (value: ArrayBuffer) => {
-      expect(value.byteLength).equals(4);
-      expect([...new Uint8Array(value)]).deep.equals([4, 8, 12, 16]);
-    },
-    flexibleSync,
+    useFlexibleSync: useFlexibleSync,
```

Comment:
When the property and the value variable share the name, you can omit the explicit set.
```suggestion
    useFlexibleSync,
```